sql/telemetry/client_ltv/view.sql

```diff
@@ -0,0 +1,20 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.client_ltv`
+AS
+SELECT
+  * EXCEPT (
```

Comment:
I noticed that we're missing normalized tagged searches LTV. Want to add that to the other query so it's available here?

Reply:
How so? It looks to be there: https://github.com/mozilla/bigquery-etl/pull/1202/files/0906389fa0d5dbb728763892e74fe8a181bddad1#

Reply:
Down below that: https://github.com/mozilla/bigquery-etl/blob/master/sql/revenue_derived/client_ltv_v1/query.sql#L246-L267

Looks like future tagged search is there, but not current.

Reply:
ah, I see. will fix!